pkg/flag/cache_flags.go

```diff
@@ -25,7 +25,7 @@ var (
 		Name:       "cache-backend",
 		ConfigName: "cache.backend",
 		Default:    "fs",
-		Usage:      "cache backend (e.g. redis://localhost:6379)",
+		Usage:      "[EXPERIMENTAL] cache backend (e.g. redis://localhost:6379)",
```

Comment:
wdyt?
```suggestion
		Values: xstrings.ToStringSlice([]cache.Type{
			cache.TypeMemory,
			cache.TypeFS,
			cache.TypeRedis,
		}),
		Usage:      "[EXPERIMENTAL] cache backend (e.g. redis://localhost:6379)",
```

Reply:
It accepts a Redis URL, like `redis://localhost:6379`.

Reply:
hm.. you are right.